testing/functional/tests/cypress/integration/student/02-login-page.spec.ts

```diff
@@ -0,0 +1,139 @@
+import WelcomeObject from "../../page-objects/student-objects/WelcomeObject"
+import Login from "../../page-objects/student-objects/02-login-object"
+
+describe("Login Page", () => {
+
+    const Welcome = new WelcomeObject()
```

Comment:
Please declare local variables using camelCase 😉

Reply:
Yes changes has been done, local variable defined in camelCase.